app/models/state_file_nj_intake.rb

```diff
@@ -194,5 +194,7 @@ def disqualifying_eligibility_rules
       health_insurance_eligibility: "ineligible"
     }
   end
+
+  def validate_state_specific_w2_requirements(w2); end
```

Comment:
[dust] A comment calling out that we're intentionally overriding the base class might be helpful, but then again maybe that should be obvious

Reply:
I was hoping that would be clear from the test for this - ie, that test will fail without this line, but could go either way here